tests/spec/core/webidl.html

```diff
@@ -505,6 +505,12 @@ <h2>Dictionaries</h2>
       <pre id='dict-basic' class='idl'>
         dictionary SuperStar {};
       </pre>
+      <p>
```

Comment:
I prefer if you kept the HTML in the other test file. Makes the tests easier to work with.

Reply:
means u want me to use 
```js
const body = `
  <pre class='webidl' id='dict-webidl'>
        dictionary Test {};
  </pre>
`;
const ops = makeStandardOps(null, body);
const doc = await makeRSDoc(ops);
const target = doc.getElementById("dict-webidl");
```
in `webidl-spec.js`?